content-module2.md

```diff
@@ -1036,7 +1106,10 @@ it("should CRUD users", async () => {
 
 </details>
 
-### Dependency patching
+# Dev notes
+
+<details>
+<summary>Dependency patching</summary>
```

Comment:
move this up as well?

Reply:
This information is not for the person who has to go through the course. I explained what they needed to do in `Class validator & Class Transformer` > `Intro` > `NOTE`.